src/colp/components/manual_correction_report.py

```diff
@@ -38,5 +46,6 @@ def display_manual_correction(self):
             st.subheader("Numbers of Manual Corrections Not Applied By Field")
             st.write(count2)
             not_applied = not_applied.sort_values(by=["field"])
+            not_applied = not_applied[not_applied.columns.tolist()[1:] + ["uid"]]
```

Comment:
You don't need to convert to list here, can index the `columns` attr in a list-like way

Reply:
It actually throws an error on my side if I don't convert to list. It will add the string 'uid' at the end of each field name.

Reply:
Ah great that makes sense, need the list for the `+` operator. Sorry about that